frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerLifeCycle.java

```diff
@@ -187,6 +187,7 @@ public void startWorker(int port, String deviceIds)
     private void attachRunner(
             ArrayList<String> argl, List<String> envp, int port, String deviceIds) {
         envp.add("LOGLEVEL=INFO");
+        envp.add("OMP_NUM_THREADS=1");
```

Comment:
Why is the value hardcoded? OMP_NUM_THREADS = 1 by default. It will be good to allow user to change this value,  my tests showed speedup on multi-gpu instances for higher value when the CPU core are available